src/components/HomepageImage/HomepageImageElements.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const ImageWrapper = styled.div`
+  background-color: orange;
+  padding: 120px 0;
+`;
+
+// export const TextWrapper = styled.div`
+
+// `;
```

Comment:
Leaving commented code is always a no-no. You can remove this code if you don't need it.

Reply:
Thanks, I'll fix it.